content/python/concepts/pip/pip.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'pip'
+Description: 'Much like how npm is used to manage code packages in JavaScript, pip also offers practical ways to install and use pre-made code.'
```

Comment:
I think this `Description` is on the right path but I think we should complete the contrast by mentioning Python, similar to the entry's introduction:

```
Description: 'Much like how npm is a package manager for JavaScript, pip is the same community-driven manager, but for Python.'